src/convolution.cpp

```diff
@@ -422,10 +422,12 @@ std::size_t ConvolutionDescriptor::ForwardGetWorkSpaceSize(Handle& handle,
     ctx.SetStream(&handle);
     ctx.DetectRocm();
 
+    const size_t workspace_size_winograd = ForwardBackwardDataGetWorkSpaceSizeWinograd(ctx);
+
     if(IsWinograd3x3SupportedAndFast(ctx))
     {
-        MIOPEN_LOG_I2(0);
-        return 0;
+        MIOPEN_LOG_I2(workspace_size_winograd);
+        return workspace_size_winograd;
```

Comment:
3x3 Winograd didn't need any workspace, is it?

Reply:
Yes, but you can guarantee that it's faster than MultiPass?

Reply:
@Kirpich30000 We know that when when 3x3 winograd is really fast, it is faster than all other algorithms. Therefore all other algorithms are skipped. However we are still trying _all_ applicable Winograds, and the one that needs workspace may win.